doc/whats_new.rst

```diff
@@ -15,6 +15,13 @@ Changelog
 New features
 ............
 
+   - Added two functions for mutual information estimation:
+     :func:`feature_selection.mutual_info_classif` and 
+     :func:`feature_selection.mutual_info_regression`. These functions can be
+     used in :class:`feature_selection.SelectKBest` and
+     :class:`feature_selection.SelectPercentile`, which now accept callable
+     returning only `scores`. By `Nikolay Mayorov`_.
```

Comment:
I'm totally up for it, but to be precise I don't think I keep any of the original code. Should I just write `... and Andrea Bravi`?